docs/aws/audit/cloudtrailmonitoring/rules/cloudtrail_enabled.mdx

```diff
@@ -23,6 +23,209 @@ HIPAA, PCIDSS, GDPR, CISAWS, CBP, NIST, SOC2, ISO27001, AWSWAF, NISTCSF, FedRAMP
+
+2. List all trails: Use the following command to list all trails in your AWS account:
+
+    ```
+    aws cloudtrail describe-trails
+    ```
+
+    This command will return a JSON output with details of all trails.
+
+3. Check if CloudTrail is enabled in all regions: For each trail in the output, check the 'IsMultiRegionTrail' field. If it is set to 'true', then CloudTrail is enabled for all regions for that trail. If it is 'false', then CloudTrail is not enabled for all regions. You can use the following command to check this:
+
+    ```
+    aws cloudtrail get-trail-status --name TrailName
+    ```
+
+    Replace 'TrailName' with the name of your trail. This command will return a JSON output with the trail status. Check the 'IsMultiRegionTrail' field in the output.
+
+4. Repeat the process: Repeat step 3 for all trails in your AWS account. If any trail has 'IsMultiRegionTrail' set to 'false', then CloudTrail is not enabled for all regions for that trail.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install the necessary Python libraries: Before you start, you need to install the AWS SDK for Python (Boto3) in your environment. You can do this using pip:
+
+   ```
+   pip install boto3
+   ```
+
+2. Import the necessary libraries and set up AWS credentials: In your Python script, you need to import the Boto3 library and set up your AWS credentials (Access Key ID and Secret Access Key). You can do this using the AWS CLI or by setting them directly in your script:
+
+   ```python
+   import boto3
+
+   aws_access_key_id = 'YOUR_ACCESS_KEY'
+   aws_secret_access_key = 'YOUR_SECRET_KEY'
+   ```
+
+3. Create a CloudTrail client and check the trail settings: Next, you need to create a CloudTrail client using Boto3 and check the trail settings for all regions. You can do this using the `describe_trails` and `get_trail_status` methods:
+
+   ```python
+   cloudtrail = boto3.client('cloudtrail', aws_access_key_id=aws_access_key_id, aws_secret_access_key=aws_secret_access_key)
+
+   trails = cloudtrail.describe_trails()
+   for trail in trails['trailList']:
+       status = cloudtrail.get_trail_status(Name=trail['Name'])
+       if not status['IsMultiRegionTrail']:
+           print(f"Trail {trail['Name']} is not enabled for all regions.")
+   ```
+
+4. Analyze the results: The script will print the names of all trails that are not enabled for all regions. If no trails are printed, then CloudTrail is enabled for all regions. If some trails are printed, then those trails are not enabled for all regions and you have detected a misconfiguration.
+</Accordion>
+
+</AccordionGroup>
+</Tab>
+
 <Tab title='Remediation'>
 ### Remediation
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>